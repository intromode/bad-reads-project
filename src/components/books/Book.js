import React from 'react';
import PropTypes from 'prop-types';

function Book({ book }) {
  const { title, author, imageUrl } = book;
  return (
    <>
      <p>{title}</p>
      <p>{author}</p>
      <img src={imageUrl}/>
    </>
  );
}

Book.propTypes = {
  book: PropTypes.object
};

export default Book;

