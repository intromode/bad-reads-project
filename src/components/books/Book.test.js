import React from 'react';
import { shallow } from 'enzyme';
import Book from './Book';

describe('App component', () => {
  it('renders a App', () => {
    const wrapper = shallow(<Book book={{title: 'hey', author: 'you', imageUrl: 'fjsdklfj'}} />);
    expect(wrapper).toMatchSnapshot();
  });
});
