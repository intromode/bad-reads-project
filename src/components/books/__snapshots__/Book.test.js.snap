// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders a App 1`] = `
<Fragment>
  <Link
    to="/hey"
  >
    <p>
      hey
    </p>
  </Link>
  <p>
    you
  </p>
  <img
    src="fjsdklfj"
  />
</Fragment>
`;
